### PR TITLE
Web3j fix android

### PR DESCRIPTION
**Pull request by shashankiitbhu**
### What does this PR do?
*required*

### Where should the reviewer start?
*required*

### Why is it needed?
*required*

## Checklist

- [x] I've read the contribution guidelines.
- [x] I've added tests (if applicable).
- [x] I've added a changelog entry if necessary.

### CODE REVIEW

File: gradle/publish/build.gradle [hunk 1]
```diff
@@ -81,10 +81,9 @@ nexusStaging {
...
-    if (signingKey.exists()) {
-        useInMemoryPgpKeys(signingKey.getText('UTF-8'), System.getenv('GPG_PASSPHRASE'))
-    }
-}
+//signing {
+//    sign publishing.publications.maven
+//    def gpgSecretKey = System.getenv('OSSRH_GPG_SECRET_KEY')
+//    def password = System.getenv('OSSRH_GPG_SECRET_KEY_PASSWORD')
+//    useInMemoryPgpKeys(gpgSecretKey, password)
+//}
```
Comment (gtebrean) [inline gradle/publish/build.gradle:89]: uncomment this else the publication will on git

**Review by gtebrean — COMMENTED** [review]
few comments

File: crypto/src/test/java/org/web3j/crypto/Secp256k1JWKTest.java [hunk 2]
```diff
@@ -127,7 +128,9 @@ public void testPrivateJWK() throws IOException {
 
     @Test
     public void testPublicJWK() throws IOException {
-        String jwk = "build/resources/test/" + "jwk/Public256k1JWK.json";
+//        String jwk = "build/resources/test/" + "jwk/Public256k1JWK.json";
```
[the remark this answers is not in the document]
Comment (shashankiitbhu) [reply crypto/src/test/java/org/web3j/crypto/Secp256k1JWKTest.java:131]: ok

File: gradle/publish/build.gradle [hunk 3]
```diff
@@ -2,33 +2,33 @@ apply plugin: 'maven-publish'
...
+//task testsJar(type: Jar) {
+//    archiveClassifier = 'tests'
+//    from sourceSets.test.output
+//}
 
-javadoc { options.encoding = 'UTF-8' }
+//javadoc { options.encoding = 'UTF-8' }
 
-artifacts { archives sourcesJar, javadocsJar, testsJar }
+//artifacts { archives sourcesJar, javadocsJar, testsJar }
```
[the remark this answers is not in the document]
Comment (shashankiitbhu) [reply gradle/publish/build.gradle:22]: ok

**Comment by gtebrean** [comment]
From what I see build on this PR is not executed at all, only the DCO takes place, please check why this is happening, the build script is in the same folder where is the release script. You can find in this PR all the jobs which needs to be executed https://github.com/hyperledger-web3j/web3j/pull/2124

**Pull request merged**